element_animal/genotyping.py

```diff
@@ -89,13 +89,14 @@ class Father(dj.Part):
         definition = """
         -> master
         ---
-        -> subject.Subject
+        -> [nullable] subject.Subject.proj(father="subject")
```

Comment:
Perhaps we don't need to make this nullable.  If null, we can just not insert into the Part table.

Reply:
Good point, I'll make that adjustment